lib/logger.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+import json
+
+from .settings import is_development
```

Comment:
why not use an existing logging solution?

Reply:
some library

Reply:
There is no "built-in" solution and a third-party directory will be too heavy at this point because we still have no third-party dependency.